#!/usr/bin/python3
def raise_exception_msg(message=""):
    """Raise a NameError excption nwith a error"""
    raise NameError(message)
